dotcom-rendering/src/components/FrontCard.tsx

```diff
@@ -52,6 +52,7 @@ export const FrontCard = (props: Props) => {
 		branding: trail.branding,
 		slideshowImages: trail.slideshowImages,
 		showLivePlayable: trail.showLivePlayable,
+		showMainVideo: trail.showMainVideo,
```

Comment:
Unfortunately this skews the logic a bit! If the main media image is above 480px and `showMainVideo` flag is off in the fronts tool:
* `isPlayableMediaCard` will be false 
* [`getMedia` ](https://github.com/guardian/dotcom-rendering/blob/d5a18dd3ee4addc7101667afc8690955ec4417a1/dotcom-rendering/src/components/Card/Card.tsx#L196-L226) will return `picture`(assuming there is an `imageUrl`)
* but [`showPlayIcon`](https://github.com/guardian/dotcom-rendering/blob/d5a18dd3ee4addc7101667afc8690955ec4417a1/dotcom-rendering/src/components/Card/Card.tsx#L382) will be true
* and we end up in this case where [`CardPicture` will be rendered with play icon](https://github.com/guardian/dotcom-rendering/blob/d5a18dd3ee4addc7101667afc8690955ec4417a1/dotcom-rendering/src/components/Card/Card.tsx#L480-L495)


| fronts tool | DCR |
|-------------|-----|
| ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/b36e5115-ef0e-4b0f-9e02-1790c6619f7f) | ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/f3319989-12cd-46d5-b789-7f57a1afe7f2) |

There is indeed this trade-off of passing down `showMainVideo` to the `Card` but I think in this case it's simpler to do it this way because `isPlayableMediaCard` determines two things: 

1. What the media type will be
2. Whether we show a play icon in a video that has an image smaller than 480px

Let me know what you think!

Reply:
Oh yeah, definitely!